faq/faq.rst

```diff
@@ -96,3 +96,8 @@ Can I run Citus on Heroku or Amazon RDS?
 ----------------------------------------
 
 At this time Heroku and Amazon do not support running Citus directly on top of Heroku PostgreSQL or Amazon RDS. It is up to them if/when they enable the Citus extension. If you are looking for something similar, `Citus Cloud <https://www.citusdata.com/product/cloud>`_ is our database-as-a-service which we fully manage for you. It runs on top of AWS (like both RDS and Heroku PostgreSQL) and should provide a very similar product experience, with the addition of Citus' horizontal scaling.
+
+Can I shard by schema on Citus for multi-tenant applications?
+-------------------------------------------------------------
+
+It turns out that while storing each tenant's information in a separate schema can be an attractive way to start when dealing with tenants, it leads to problems down the road. In Citus, tenants share database tables instead. To learn more about the reason for this design, see our article `Lessons learned from PostgreSQL schema sharding <https://www.citusdata.com/blog/2016/12/18/schema-sharding-lessons/>`_.
```

Comment:
Instead of "tenants share database tables", may be "In Citus we partition by the tenant_id, and a shard can contain data from several tenants".
Otherwise Looks good.